paddlets/datasets/tsdataset.py

```diff
@@ -1482,7 +1482,7 @@ def set_column(
         self,
         column: Union[str, int],
         value: Union[pd.Series, str, int],
-        type: str = 'known_cov'
+        type: Optional[str] = 'known_cov'
```

Comment:
Line1485 should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional